### DISCUSSION TITLE
Would you like to be the new owner of logback-android?

### DISCUSSION THREAD

**Discussion by tony19** [tony19/logback-android#329] [Announcements]
As much as I love working on this project, I have recently moved on to new endeavors in my career, which unfortunately means that I can no longer give this library the attention it deserves.

If you are a passionate developer who enjoys working on open-source projects and would like to take over as the new maintainer, I would love to hear from you!

**Comment by mrcsxsiq** [comment]
Owner is a strong word, but I would like to contribute to this project. I recently discovered it and I'm loving it. It's helping me a lot.

You can check out my GitHub profile. I have a few open-source projects and I'm looking for a new open-source project to keep my efforts going on. I'm trying to get something new in telemetry and observability and I think logback-android is aligned with it.

If you consider me one of the maintainers, I would be honored.

Let's keep in touch.

**Reply by tony19** [reply]
Happy to have you. Added you as a maintainer. Thanks! 🙏

**Reply by Malachosky17** [reply]
@mrcsxsiq have you actively supported this project? I don't see releases post CY23. Do you have any updates for developers wanting to rely on this implementation of logback?
